src/styles/blocks/catalog.scss

```diff
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition-duration: 300ms;
```

Comment:
don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
![image](https://user-images.githubusercontent.com/91411692/223092756-9b99108e-339b-4521-a0be-3ff3ea52d22d.png)
& will be better if you create vars for transition duration & colors
```suggestion
    transition: transform 300ms, color 300ms;
```